dspace/config/spring/api/core-services.xml

```diff
@@ -143,70 +148,206 @@
+     joined together with this separator.
+     If the related item has:
+        person.identifier.lastname = Smith
+        person.identifier.firstname = John
+     Then the original item will have, in this case:
+        dc.contributor.author = Smith, John -->
+    <bean class="org.dspace.content.virtual.Concatenate" id="projectPerson_author">
+        <property name="fields">
+            <util:list>
+                <value>person.identifier.lastname</value>
+                <value>person.identifier.firstname</value>
+            </util:list>
+        </property>
+        <property name="separator">
+            <value>, </value>
+        </property>
+    </bean>
 
     <util:map id="isOrgUnitOfProjectMap">
         <entry key="project_contributor_other" value-ref="projectOrgUnit_other"/>
     </util:map>
-    <util:list id="projectOrgUnit_other">
-        <value>orgunit.identifier.name</value>
-    </util:list>
+    <!-- This Collected bean will take all the values of each metadata field defined in the list and it'll
+         create a list of virtual metadata fields defined by the map in which it's defined.
+         In this case it'll create a list of project.contributor.other virtual metadata fields that hold
+         the value of orgunit.identifier.name, each of them in a separate metadatavalue
+         If the related item has:
+            orgunit.identifier.name = Generic name
+            orgunit.identifier.name = Even more generic name
+         Then the original item will have, in this case:
+            project.contributor.other = Generic name
+            project.contributor.other = Even more generic name -->
+    <bean class="org.dspace.content.virtual.Collected" id="projectOrgUnit_other">
+        <property name="fields">
+            <util:list>
+                <value>orgunit.identifier.name</value>
+            </util:list>
+        </property>
+    </bean>
 
     <util:map id="isOrgUnitOfPersonMap">
         <entry key="person_contributor_other" value-ref="personProject_other"/>
     </util:map>
-    <util:list id="personProject_other">
-        <value>orgunit.identifier.name</value>
-    </util:list>
+    <bean class="org.dspace.content.virtual.Collected" id="personProject_other">
+        <property name="fields">
+            <util:list>
+                <value>orgunit.identifier.name</value>
+            </util:list>
+        </property>
+    </bean>
 
     <util:map id="isJournalVolumeOfIssueMap">
-        <entry key="journalvolume_identifier_volume" value-ref="issueVolume_volume"/>
+        <entry key="journalvolume.identifier.volume" value-ref="issueVolume_volume"/>
+        <entry key="journal.identifier.issn" value-ref="volumeJournal_issn_related"/>
     </util:map>
-    <util:list id="issueVolume_volume">
-        <value>journalvolume.identifier.volume</value>
-    </util:list>
+    <!-- This Collected bean will take all the values of each metadata field defined in the list and it'll
```

Comment:
Similar to above comment, it'd be good to copy this detailed example into the `Collected` bean's Javadocs.  Again, we likely don't need detailed examples for every instance of `Collected` in this config file, as long as we can link to Javadocs for more info.

Reply:
I've moved the general comments to the JavaDoc of the class, and only left the specific comments explaining the use case for that configuration